### DISCUSSION TITLE
Advice on how to fix the issue where the IDE is claiming Biome doesn't support typescript.

### DISCUSSION THREAD

**Discussion by salieflewis** [biomejs/biome#420] [Q&A]
I've set my manual configuration to be biome, but for some reason my IDE doesn't recognize Biome when I go to configure the default formatter.

`Extension 'Biome' is configured as formatter but it cannot format 'TypeScript'-files`

Using VSCode. I've also tried to install Biome manually and via the extensions marketplace.

**Comment by ematipico** [comment]
That's weird, it's been a while since I've seen this. Did you open an ignored file? Did you try to restart the editor?

**Reply by salieflewis** [reply]
What do you mean by open an ignored file?

I've tried to restart the editor yes.

**Reply by ematipico** [reply]
> What do you mean by open an ignored file?
> 

A file that you ignored in the configuration.

I think it's best to create and share a minimal repository, so we can triage the problem

**Comment by GabenGar** [comment]
Are you using a multi-root workspace, with a project with biomejs nested somewhere inside it, by any chance?

**Reply by akre54** [reply]
This issue is happening for me, and yes I can confirm that I get the error when I have our monorepo open (with multi-root projects, one of which has biome), but not when I have VSCode just open the single repo that uses biome

**Reply by ematipico** [reply]
The extension doesn't have good support for workspaces, there's an issue open. Any help to add support to workspaces would be appreciated
